elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -50,6 +50,9 @@ protected function setup_constants() {
 	protected function setup_hooks() {
 		add_filter( 'ep_index_name', [ $this, 'filter__ep_index_name' ], PHP_INT_MAX, 3 ); // We want to enforce the naming, so run this really late.
 
+		// Override default per page value set in elasticsearch/elasticpress/includes/classes/Indexable.php
+		add_filter( 'ep_bulk_items_per_page', function() { return 500; } );
```

Comment:
To keep this aligned with the other filters we have, we should switch this out to a function on the class (I mostly suggested an anonymous function here as a PoC).